chainercv/datasets/sbd/sbd_instance_segmentation_dataset.py

```diff
@@ -21,7 +21,7 @@ def _check_available():
             'SciPy is not installed in your environment,',
```

Comment:
the comma at the end of this line causes this error.
I'm not sure, but `RuntimeWarning` does not matter according to https://github.com/chainer/chainercv/blob/master/chainercv/extensions/vis_report/detection_vis_report.py#L17-L22

Reply:
I see. I updated.